src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1592409240.inc.php

```diff
@@ -0,0 +1,26 @@
+<h1>Build #1592409240</h1>
```

Comment:
(issue description) Why is it only sometimes crashing?

Reply:
to be honest, I have no idea why it crashes only sometimes. But is that relevant? The missconfiguration of the field leads to a broken method in the TAdb that attempts to load a 'TdbParent' class - which does not exists.